### PR TITLE
Darkmode fix for elements that failed to change colour

### LINKED ISSUE #147: Some elements fail to change color in dark mode
#### I am submitting a -
* [] **Bug Report**

#### Describe your bug/feature -
When I switch the dark mode  on, I expect that everything inverts color according to the dark theme. But it fails in the case of multiplexer documentation.
#### Current behaviour -
Second input fails to change color.
#### Expected behaviour -
It should change color.
#### Screenshots (if relevant) -
before
after

### PR DESCRIPTION
**Pull request by schezfaz**
Fixes #147 

#### Changes done:
- [x] Created a new .css file: styles/darkTheme.css to override [docsify dark-theme css file](//unpkg.com/docsify/themes/dark.css) in order to alter the colour of the elements that failed to transition in the dark-mode.
- [x] In index.html added a provision to override the stylesheet selected, based on whether it is light or dark mode view.

#### Screenshots of changes:
List of all pages that failed to change colour in dark mode:
1. Standard Gates: all truth tables
2. Decodes and Plexers: all truth tables
3. Sequential Elements: all truth tables
4. Miscellaneous: all truth tables

Post fix, they change their colour according to the theme:
image

Sample preview link after checks complete:  https://github.com/CircuitVerse/CircuitVerseDocs/pull/226/checks?check_run_id=689902655

#### ✅️ By submitting this PR, I have verified the following

- [x] Checked to see if a similar PR has already been opened 🤔️
- [x] Reviewed the contributing guidelines 🔍️
- [x] Sample preview link added (add a link from the checks tab after checks complete)
- [ ] Tried Squashing the commits into one

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -17,13 +17,16 @@
...
+        <link id="stylesheet_override" rel="stylesheet" type="text/css" href="styles/theme.css">
 
  <script>
 
 	var url =  "//unpkg.com/docsify/themes/dark.css";
     	var url2 = "//unpkg.com/docsify/themes/vue.css";
+		var override2 = "styles/theme.css";
+		var override = "styles/darkTheme.css";
     	var stylesheet = document.getElementById("stylesheet");
+		var stylesheet_override = document.getElementById("stylesheet_override");
```
Comment (YashKumarVerma) [inline docs/index.html:29]: I think we need to fix the indentation here.

File: docs/styles/darkTheme.css [hunk 2]
```diff
@@ -0,0 +1,3 @@
+.markdown-section td {
+    background: #3F3F3F;
+}
```
Comment (YashKumarVerma) [inline docs/styles/darkTheme.css:3]: new line at the end is always recommended.

File: docs/index.html (hunk 1 above)
Comment (schezfaz) [reply docs/index.html]: done!

File: docs/styles/darkTheme.css (hunk 2 above)
Comment (schezfaz) [reply docs/styles/darkTheme.css]: sure, made the requested changes.

File: docs/index.html [hunk 3]
```diff
@@ -61,16 +59,15 @@
...
+				    var footer = [
 					'<header>',
 				       	'<span><label class="switch"><input title="Toggle Dark theme" onclick="setStyleSheet();" type="checkbox"><span class="slider round"></span></label></span>',
 					'</header>'
-				      ].join('');
+				    ].join('');
 
-				      hook.afterEach(function(html) {
-					return footer + html;
-				      });
```
Comment (YashKumarVerma) [inline docs/index.html:73]: this was indented right

File: docs/index.html [hunk 4]
```diff
@@ -61,16 +59,15 @@
...
+				    hook.afterEach(function(html) {
+						return footer + html;
+				    });
```
Comment (YashKumarVerma) [inline docs/index.html:70]: extra spaces before return

File: docs/index.html [hunk 5]
```diff
@@ -84,16 +81,15 @@
 			noCompileLinks: ['/#'],
 			auto2top: true,
 			pagination: {
-                             previousText: 'Previous',
-                             nextText: 'Next',
+							previousText: 'Previous',
+							nextText: 'Next',
```
Comment (YashKumarVerma) [inline docs/index.html:85]: excessive spacing

File: docs/index.html [hunk 6]
```diff
@@ -84,16 +81,15 @@
...
                        	}
  
 		}
 	</script>
-	 
 	<script src="//unpkg.com/docsify@4.7.1/lib/docsify.min.js"></script>
 	<script src="//unpkg.com/docsify/lib/plugins/search.min.js"></script>
-    	<script src="//unpkg.com/docsify-pagination/dist/docsify-pagination.min.js"></script>
```
Comment (YashKumarVerma) [inline docs/index.html:96]: this was right ? all script tags were aligned right ?

File: docs/index.html (hunk 3 above)
Comment (schezfaz) [reply docs/index.html:73]: Yes, however I indented it with respect to the rest of the scope as follows. Should I revert this change?

image

File: docs/index.html (hunk 5 above)
Comment (schezfaz) [reply docs/index.html:85]: fixing this!

File: docs/index.html (hunk 6 above)
Comment (schezfaz) [reply docs/index.html:96]: yes, they were but there was an extra space between the first script tag and the next, in the <body> tag, so i just eliminated that. Should I revert it?

**Comment by YashKumarVerma** [comment]
Thanks a lot for your contributions.

**Comment by schezfaz** [comment]
Thankyou for your guidance!

**Comment by YashKumarVerma** [comment]
@Shivansh2407 here.

**Pull request merged**